test/scripts/jenkins_xpack.sh

```diff
@@ -27,11 +27,11 @@ if [[ -z "$CODE_COVERAGE" ]] ; then
   echo ""
   echo ""
 
-  # echo " -> Running jest integration tests"
-  # cd "$XPACK_DIR"
-  # node scripts/jest_integration --ci --verbose
-  # echo ""
-  # echo ""
+  echo " -> Running jest integration tests"
```

Comment:
Re-enables jest integration tests on CI for x-pack as described in https://github.com/elastic/kibana/issues/67108#issuecomment-631689119